.circleci/config.yml

```diff
@@ -24,7 +40,39 @@ jobs:
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-
+    - run: git diff --exit-code
+    - prometheus/store_artifact:
+        file: prombench
+    - prometheus/store_artifact:
+        file: scaler
+    - prometheus/store_artifact:
+        file: fake-webserver
+   
+  publish_images:
+    description: |
+      Build and publish prombench image from the master branch.
+    docker:
+    - image: circleci/golang
+    steps:
+    - prometheus/setup_build_environment
+    - publish_images:
+        dockerfile_path: ./cmd/prombench/Dockerfile
+        container_image_name: prombench
+    - publish_images:
+        dockerfile_path: ./tools/amGithubNotifier/Dockerfile
+        container_image_name: amgithubnotifier
```

Comment:
For later but it would good for consistency to include this binary in `.promu.yml`.

Reply:
Will do it in another PR